BrainPortal/db/seeds_dev.rb

```diff
@@ -806,7 +806,7 @@ def seeds_dev_tmp_file(content)
 User.all.each do |user|
   groups = [ user.own_group ] + user.groups.where( :type => 'WorkGroup' ).all
   groups.each do |group|
-    Bourreau.all.each do |bourreau|
+    [ main_bourreau, pember_bourreau, longbourne_bourreau ].each do |bourreau|
```

Comment:
We discovered this change was necessary because it made the seeding work even when a Bourreau was already present; however now that I think of it we should go over the rest of this file and also fix other places where we have `Bourreau.all` or `User.all` etc. Let's make the seeding process only affect its own objects.